plutus-tx/test/Lift/Spec.hs

```diff
@@ -15,16 +15,28 @@ import qualified Language.PlutusTx.Builtins as Builtins
 import           Language.PlutusTx.Code
 import qualified Language.PlutusTx.Lift     as Lift
 
+-- this module does lots of weird stuff deliberately
+{-# ANN module ("HLint: ignore"::String) #-}
+
 Lift.makeLift ''MyMonoData
 Lift.makeLift ''MyMonoRecord
 Lift.makeLift ''MyPolyData
 
-newtype NestedRecord = NestedRecord { unNested :: Maybe (Int, Int) }
+data NestedRecord = NestedRecord { unNested :: Maybe (Int, Int) }
```

Comment:
It does, but that wasn't what I wanted to test here. i.e. I want to test this makes nested datatypes, rather than doing the newtype-equals-underlying-type thing.